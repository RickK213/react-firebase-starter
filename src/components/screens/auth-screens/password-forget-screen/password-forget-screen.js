import React from 'react';
import { PasswordForgetForm } from './password-forget-form/password-forget-form';

export const PasswordForgetScreen = () => {
  return (
    <div>
      <h2>Forgot Password?</h2>
      <PasswordForgetForm />
    </div>
  );
};
