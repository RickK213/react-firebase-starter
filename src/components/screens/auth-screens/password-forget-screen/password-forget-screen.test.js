/* global React, create */
import { StaticRouter } from 'react-router-dom';
import { PasswordForgetScreen } from './password-forget-screen';

describe('<PasswordForgetScreen />', () => {
  it('should render correctly', () => {
    const component = create(
      <StaticRouter context={{}}>
        <PasswordForgetScreen />
      </StaticRouter>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
