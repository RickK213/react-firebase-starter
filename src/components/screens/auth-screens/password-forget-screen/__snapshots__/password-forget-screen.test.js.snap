// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PasswordForgetScreen /> should render correctly 1`] = `
<div>
  <h2>
    Forgot Password?
  </h2>
  <form
    autoComplete="off"
    onSubmit={[Function]}
    style={
      Object {
        "width": "360px",
      }
    }
  >
    <input
      autoComplete="off"
      name="email"
      onChange={[Function]}
      placeholder="Email Address"
      style={
        Object {
          "display": "block",
          "marginBottom": "1rem",
          "padding": ".5rem",
          "width": "100%",
        }
      }
      type="email"
      value=""
    />
    <button
      disabled={true}
      style={
        Object {
          "padding": ".5rem",
          "textTransform": "uppercase",
        }
      }
      type="submit"
    >
      Reset My Password
    </button>
  </form>
</div>
`;
